### PR TITLE
Add API endpoint allowing user to sign developer agreement.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @diox @ngokevin

### CODE REVIEW

File: mkt/account/tests/test_views.py [hunk 1]
```diff
@@ -942,3 +943,36 @@ def test_remove_group_fail_not_member(self):
...
+class TestTOSView(RestOAuth):
+    fixtures = fixture('user_2519')
+
+    def setUp(self):
+        super(TestTOSView, self).setUp()
+        self.url = reverse('api-v2:account-tos')
+        self.user = UserProfile.objects.get(pk=2519)
+
+    def test_verbs(self):
+        self._allowed_verbs(self.url, ('post'))
```
Comment (diox) [inline mkt/account/tests/test_views.py]: add a test for cors as well

File: docs/api/topics/accounts.rst [hunk 2]
```diff
@@ -232,3 +232,20 @@ carrier/region pairs.
         [
             '*'
         ]
+
+
+Sign Developer Agreement
+========================
+
+.. http:post:: /api/v2/account/tos/
```
Comment (diox) [inline docs/api/topics/accounts.rst]: nit: should it live in /tos/ if it's just the developer terms of service ?

**Comment by diox** [comment]
r+wc

File: docs/api/topics/accounts.rst (hunk 2 above)
Comment (chuckharmston) [inline docs/api/topics/accounts.rst]: /devtos/ better?

File: mkt/account/tests/test_views.py (hunk 1 above)
Comment (chuckharmston) [inline mkt/account/tests/test_views.py]: :heavy_check_mark:

File: docs/api/topics/accounts.rst (hunk 2 above)
Comment (chuckharmston) [inline docs/api/topics/accounts.rst]: :heavy_check_mark:

**Pull request merged**